bettercli/cl/input.py

```
@@ -0,0 +1,236 @@
+            "fg-color": c.ANSIColors.reset,
+        }),
+        "selector": Dict({
+            "cursor": Dict({
+                "fg-color": c.ANSIColors.cyan,
+            }),
+            "not cursor": Dict({
+                "fg-color": c.ANSIColors.reset,
+            }),
+        })
+    })
+    def __init__(self, prompt: str, suggestions: dict[str, str] = None, *, style: 'INPUT_STYLES' = {}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)